client/package.json

```diff
@@ -4,12 +4,13 @@
   "private": true,
   "dependencies": {
     "bower": "^1.7.9",
-  "grunt": "^1.0.1",
-  "gulp": "^3.9.1",
+    "grunt": "^1.0.1",
+    "gulp": "^3.9.1",
     "@azure/communication-administration": "1.0.0-beta.4",
     "@azure/communication-calling": "1.9.1-beta.1",
     "@azure/communication-calling-effects": "1.0.0-alpha.2",
-    "@azure/communication-react":"1.4.1-beta.1",
+    "@azure/communication-react": "1.4.1-beta.1",
+    "@azure/communication-chat": "1.3.0",
```

Comment:
do we need chat?

Reply:
yes otherwise it is giving error while npm build, and we need to install it manually.